config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-presubmits.yaml

```diff
@@ -0,0 +1,33 @@
+presubmits:
+  kubernetes/cloud-provider-vsphere:
+  - name: pull-cloud-provider-vsphere-test
+    agent: kubernetes
+    context: pull-cloud-provider-vsphere-test
+    branches:
+    - master
+    always_run: true
+    rerun_command: "/test pull-cloud-provider-vsphere-test"
+    trigger: "(?m)^/test( all| pull-cloud-provider-vsphere-test),?(\\s+|$)"
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
```

Comment:
It doesn't look like `make test` uses bazel at all, so probably neither of the bazel presets should be enabled.